gui/wxpython/startup/utils.py

```diff
@@ -55,6 +58,47 @@ def get_possible_database_path():
     return path
 
 
+def create_database_directory():
+    """Creates the standard GRASS GIS directory.
+
+    Creates database directory named grassdata in the standard location 
+    according to the platform.  
+    
+    Returns the new path as a string or None if nothing was found or created.
+    """
+    home = os.path.expanduser('~')     
+   
+    # Candidate for case independent "grassdata" for Linux and macOS
+    if sys.platform.startswith('linux') or sys.platform.startswith('darwin'):
+        candidate = os.path.join(home, "grassdata")
+    # Candidates for case independent "grassdata" in other dirs (Windows)
+    else:  
+        candidate = os.path.join(home, "Documents", "grassdata")
+        
+    # Create "grassdata" directory                          
+    try:
+        os.mkdir(candidate)
+        return candidate
+    except OSError:
+        pass
+
+    # Temporary "grassdata" directory     
```

Comment:
Say something like "Create a temporary "grassdata" directory if GRASS is running in some special environment and the standard directories cannot be created which might be the case in some "try out GRASS" use cases."

(Note that a line should be 88 characters or less.)